js/src/test/scala/cats/tests/FutureTests.scala

```diff
@@ -47,4 +47,5 @@ class FutureTests extends CatsSuite {
 
   checkAll("Future[Int]", MonadErrorTests[Future, Throwable].monadError[Int, Int, Int])
   checkAll("Future[Int]", ComonadTests[Future].comonad[Int, Int, Int])
+  checkAll("Future", MonadRecTests[Future].monadRec[Int, Int, Int])
```

Comment:
@johnynek I don't believe so—it [looks like](https://github.com/typelevel/cats/blob/master/laws/src/main/scala/cats/laws/FlatMapRecLaws.scala#L14) it just checks for consistency with `flatMap`.